### ISSUE TITLE
Cannot read property 'scale' of undefined （按需引入报错）

### ISSUE THREAD

**Issue by ccqgithub** [apache/incubator-echarts#13804]
### Version
5.0.0

### Steps to reproduce
    import echarts from 'echarts/lib/echarts';
    import 'echarts/lib/chart/line';

    const chartNumberA = echarts.init(document.getElementById('xxx'));

    const option = {
      xAxis: {
        type: 'category',
        boundaryGap: false,
        data: ['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat', 'Sun']
      },
      yAxis: {
        type: 'value'
      },
      series: [
        {
          data: [820, 932, 901, 934, 1290, 1330, 1320],
          type: 'line',
          areaStyle: {}
        }
      ]
    };

    chartNumberA.setOption(option);

### What is expected?
希望按需引入能够正常工作。

### What is actually happening?
报错：Cannot read property 'scale' of undefined

定位：
```
    var xAxis = xAxisModel.axis;
    var yAxis = yAxisModel.axis;
    var xRawExtentInfo = xAxis.scale.rawExtentInfo;
    var yRawExtentInfo = yAxis.scale.rawExtentInfo;
```

直接全局引入`var echarts = require('echarts');`则没有这个问题。

**Comment by pissang** [comment]
Same to https://github.com/apache/incubator-echarts/issues/13764#issuecomment-739492136

**Comment by 33YANG** [comment]
this error only occur with 'line', not 'bar', why

**Comment by mrqi** [comment]
the same problem.

**Comment by resultto** [comment]
the same problem.

**Comment by George191** [comment]
same problem

**Comment by Aetherus** [comment]
same problem with scatter chart.

**Comment by tyfoan** [comment]
```
import 'echarts/lib/coord/cartesian/Grid';
import 'echarts/lib/coord/cartesian/Axis2D';
```

worked for me

**Comment by GermanBluefox** [comment]
> ```
> import 'echarts/lib/coord/cartesian/Grid';
> import 'echarts/lib/coord/cartesian/Axis2D';
> ```
> 
> I added these lines and it worked for me

That worked for me. Thank you!

**Comment by Aetherus** [comment]
I downgraded the echarts to 4.9.x and all works fine.

**Comment by BinYongSheng** [comment]
> ```
> import 'echarts/lib/coord/cartesian/Grid';
> import 'echarts/lib/coord/cartesian/Axis2D';
> ```
> 
> I added these lines and it worked for me

That worked for me.thx

**Comment by mattdamon108** [comment]
> ```
> import 'echarts/lib/coord/cartesian/Grid';
> import 'echarts/lib/coord/cartesian/Axis2D';
> ```
> 
> I added these lines and it worked for me

Thanks!

**Comment by pissang** [comment]
This bug will be fixed in 5.0.1.

And because `line`, `bar`, `scatter` can be used on other coordinate systems. We will remove `grid`  dependency from these charts so we can have minimal bundle when these charts only used on other coordinate systems.  

So since 5.0.1, we need to import the `grid` component explicitly. 

```js
import echarts from 'echarts/lib/echarts';
import 'echarts/lib/chart/line';
import 'echarts/lib/component/grid';
```
